operations/helm/charts/mimir-distributed/ci/offline/test-dnsconfig-values.yaml

```diff
@@ -0,0 +1,59 @@
+global:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+distributor:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+overrides_exporter:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+querier:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+query_frontend:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+query_scheduler:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+ruler_querier:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+ruler_query_frontend:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+ruler_query_scheduler:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "3"
+
+ruler:
+  dnsConfig:
+    options:
+      - name: ndots
+        value: "4"
```

Comment:
Could you enable the setting for the alertmanager, compactor, overrides-exporter, store-gateway, and memcached here?

Reply:
done